easybuild/easyconfigs/p/PGPLOT/PGPLOT-5.2.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CmdCp'
+
+name = 'PGPLOT'
+version = '5.2.2'
+
+homepage = 'https://sites.astro.caltech.edu/~tjp/pgplot/'
+description = """The PGPLOT Graphics Subroutine Library is a Fortran- or C-callable,
+device-independent graphics package for making simple scientific graphs. It is intended
+for making graphical images of publication quality with minimum effort on the part of
+the user. For most applications, the program can be device-independent, and the output
+can be directed to the appropriate device at run time."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['ftp://ftp.astro.caltech.edu/pub/%(namelower)s/']
+sources = ['%(namelower)s522.tar.gz']
+patches = ['%(name)s-%(version_major_minor)s.2_gfortran-deps-fixes.patch']
+checksums = [
+    {'pgplot522.tar.gz': 'a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4'},
+    {'PGPLOT-5.2.2_gfortran-deps-fixes.patch': 'b1822eb32499dc18aa4aa4fae0b31c3c64ff8abb3a243716e78addafa7e58cec'},
+]
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+dependencies = [
+    ('libpng', '1.6.40'),
+    ('zlib', '1.2.13'),
+    ('X11', '20231019'),
+]
+
+# create .so symlinks for shared libraries
+postinstallcmds = [
+    "cd %(installdir)s/lib && ln -s libcpgplot.so.%(version)s libcpgplot.so.5",
```

Comment:
I'm wondering if the `.so` could also be replaced by `SHLIB_EXT` here, I guess so?